src/py/flwr/server/state/in_memory_state.py

```diff
@@ -186,3 +193,10 @@ def unregister_node(self, node_id: int) -> None:
     def get_nodes(self) -> Set[int]:
         """Return all available client nodes."""
         return self.node_ids
+
+    def create_workload(self) -> str:
+        """Create one workload."""
+        # Create, store, and return workload ID
+        workload_id = str(random.randrange(9223372036854775808))
+        self.workload_ids.add(workload_id)
```

Comment:
Maybe validate that the workload ID is not in the set?

Reply:
sure.